### PR TITLE
Tensorflow devcontainer

### PR DESCRIPTION
**Pull request by bhack**
Add Tensorflow devcontainer

### CODE REVIEW

**Comment by Chuxel** [comment]
@bhack Let me know when you think this is to the point it should be reviewed. I assume you're still making edits like you had for the tenserflow_addons PR.

**Comment by bhack** [comment]
No this is totally independent cause it is not waiting for a upstream image. We are wrapping on the official one. So if you want you can make a pass. I suppose that pylint is enabled by default right?

File: repository-containers/github.com/tensorflow/tensorflow/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      ],
+
+	// Uncomment when using a ptrace-based debugger like C++, Go, and Rust
+	"runArgs": [ 
+		"--cap-add=SYS_PTRACE",
+		// Uncomment this to enable Nvidia support
+		//"--runtime=nvidia",
+		"--security-opt",
+		"seccomp=unconfined" ],
+	"remoteUser": "1000"
```
Comment (Chuxel) [inline repository-containers/github.com/tensorflow/tensorflow/.devcontainer/devcontainer.json:55]: I'd recommend the name of the user in the container here (which I assume is vscode).  The UID can be updated change on Linux if your local OS UID is different.

**Comment by bhack** [comment]
@Chuxel If you can try it on GPU we could merge.

**Comment by Chuxel** [comment]
Unfortunately, I don't have a Linux machine with a NVIDIA GPU in it. (My Linux laptop has an Intel GPU.) My Azure VMs aren't using GPU SKUs at the moment either unfortunately.

**Comment by bhack** [comment]
Ok let's merge we will see if somebody with GPU will open a ticket.

**Pull request merged**